components/usage-api/usage/v1/usage.proto

```diff
@@ -21,6 +21,13 @@ message ListBilledUsageRequest {
     // to specifies the end time range for this request.
     // All instances which existed ending at to will be returned.
     google.protobuf.Timestamp to = 3;
+
+    enum Ordering {
+        ORDERING_DESCENDING = 0;
```

Comment:
The 0 value will ensure that if not specified, it will be descending by default.